addons/generic-workflows/@{{ cookiecutter.project_slug }}/.github/workflows/app.yaml

```diff
@@ -0,0 +1,165 @@
+name: Application
+
+on:
+  pull_request:
+    branches: [main]
+    paths:
+      - "app/**"
+      - "deploy/app/**"
+      - ".github/workflows/app.yaml"
+  push:
+    branches:
+      branches: [main]
+    paths:
+      - "app/**"
+      - "deploy/app/**"
+      - ".github/workflows/app.yaml"
+
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+
+jobs:
+  build_push:
+    name: Build and push ${{ matrix.environment }} image
+    strategy:
+      fail-fast: true
+      matrix:
+        {%- if cookiecutter.github_environments %}
+        stack: [infra]
+        environment:
+          {%- for env in cookiecutter.environments %}
+          - @{{ env }}
+          {%- endfor %}
+        {%- else %}
+        include:
+        {%- for env, values in cookiecutter.environments.items() %}
+          - environment: @{{ env }}
+          - aws_account_id: @{{ values.aws_account_id }}
+          - aws_region: @{{ values.tf_state_region }}
+          - aws_role_name: @{{ values.aws_role_name }}
+        {%- endfor %}
```

Comment:
I suggest we use the same logic as above.

stack: infra
environment:
  - environment: sandbox
     aws_account_id: 1234567

Reply:
Fixed